src/java/org/apache/cassandra/config/AccordSpec.java

```diff
@@ -71,4 +73,62 @@ public enum TransactionalRangeMigration
     public TransactionalMode default_transactional_mode = TransactionalMode.off;
     public boolean ephemeralReadEnabled = false;
     public boolean state_cache_listener_jfr_enabled = true;
+    public final JournalSpec journal = new JournalSpec();
+
+    public static class JournalSpec implements Params
+    {
+        public int segmentSize = 32 << 20;
```

Comment:
we don't actually need to have our variables as `segment_size` as the yaml layer converts camel case to the expected naming...   this is *only* `segment_size` in yaml, same for every field